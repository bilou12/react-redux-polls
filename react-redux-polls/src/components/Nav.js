import { Link } from "react-router-dom";

const Nav = () => {
  return (
    <nav className="nav">
      <ul>
        <li>
          <Link to="/login">Login</Link>
        </li>
        <li>
          <Link to="/">Home</Link>
        </li>
        <li>
          <Link to="/add">New Pool</Link>
        </li>
        <li>
          <Link to="/leaderboard">Leaderboard</Link>
        </li>
      </ul>
    </nav>
  );
};

export default Nav;
