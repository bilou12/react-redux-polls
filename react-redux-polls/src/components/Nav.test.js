import React from "react";
import { render } from "@testing-library/react";
import { BrowserRouter as Router } from "react-router-dom";
import Nav from "./Nav";

test("Nav component renders correctly", () => {
  const { container } = render(
    <Router>
      <Nav />
    </Router>
  );
  expect(container).toMatchSnapshot();
});
