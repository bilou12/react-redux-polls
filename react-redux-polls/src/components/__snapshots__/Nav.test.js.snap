// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav component renders correctly 1`] = `
<div>
  <nav
    class="nav"
  >
    <ul>
      <li>
        <a
          href="/login"
        >
          Login
        </a>
      </li>
      <li>
        <a
          href="/"
        >
          Home
        </a>
      </li>
      <li>
        <a
          href="/add"
        >
          New Pool
        </a>
      </li>
      <li>
        <a
          href="/leaderboard"
        >
          Leaderboard
        </a>
      </li>
    </ul>
  </nav>
</div>
`;
